### ISSUE TITLE
Uncaught TypeError: Cannot read property 'classList' of null

### ISSUE THREAD

**Issue by lorismaz** [mulesoft/api-workbench#511]
1. Packages > API Workbench > Create RAML Project
2. Leave all by default
3. Uncheck last 2 options
4. Got this error along with [Uncaught TypeError: Cannot read property 'destroy' of undefined](https://github.com/mulesoft/api-workbench/issues/509)

**Atom**: 1.19.1 x64
**Electron**: 1.6.9
**OS**: Mac OS X 10.12.6
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package 0.8.46

### Stack Trace

Uncaught TypeError: Cannot read property 'classList' of null

```
At /Applications/Atom.app/Contents/Resources/app/src/pane-resize-handle-element.js:25

TypeError: Cannot read property 'classList' of null
    at HTMLElement.PaneResizeHandleElement.attachedCallback (/Applications/Atom.app/Contents/Resources/app/src/pane-resize-handle-element.js:25:45)
    at HTMLElement.PaneAxisElement.childRemoved (/Applications/Atom.app/Contents/Resources/app/src/pane-axis-element.js:111:21)
    at HTMLElement.PaneAxisElement.childReplaced (/Applications/Atom.app/Contents/Resources/app/src/pane-axis-element.js:122:12)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneAxis.module.exports.PaneAxis.replaceChild (/Applications/Atom.app/Contents/Resources/app/src/pane-axis.js:209:33)
    at Pane.module.exports.Pane.split (/Applications/Atom.app/Contents/Resources/app/src/pane.js:984:27)
    at Pane.module.exports.Pane.splitDown (/Applications/Atom.app/Contents/Resources/app/src/pane.js:972:25)
    at /packages/api-workbench/main.js:1:7652)
    at t.display (/packages/api-workbench/main.js:1:10736)
    at t.updateEverything (/packages/api-workbench/main.js:1:11759)
    at /packages/api-workbench/main.js:1:8222
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at Workspace.didChangeActivePaneItemOnPaneContainer (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:493:28)
    at WorkspaceCenter.paneContainer.onDidChangeActivePaneItem (/Applications/Atom.app/Contents/Resources/app/src/workspace-center.js:17:20)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.didChangeActiveItemOnPane (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:284:26)
    at PaneContainer.didActivatePane (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:262:18)
    at Pane.module.exports.Pane.activate (/Applications/Atom.app/Contents/Resources/app/src/pane.js:911:20)
    at Pane.module.exports.Pane.focus (/Applications/Atom.app/Contents/Resources/app/src/pane.js:290:25)
    at HTMLElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app/src/pane-element.js:51:25)
    at HTMLElement.PaneElement.attachedCallback (/Applications/Atom.app/Contents/Resources/app/src/pane-element.js:30:21)
    at HTMLElement.module.exports.PaneContainerElement.rootChanged (/Applications/Atom.app/Contents/Resources/app/src/pane-container-element.js:42:14)
    at Function.module.exports.Emitter.simpleDispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:25:20)
    at Emitter.module.exports.Emitter.emit (/Applications/Atom.app/Contents/Resources/app/node_modules/event-kit/lib/emitter.js:141:34)
    at PaneContainer.setRoot (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:138:24)
    at PaneContainer.deserialize (/Applications/Atom.app/Contents/Resources/app/src/pane-container.js:58:16)
    at WorkspaceCenter.deserialize (/Applications/Atom.app/Contents/Resources/app/src/workspace-center.js:31:30)
    at Workspace.deserialize (/Applications/Atom.app/Contents/Resources/app/src/workspace.js:416:42)
    at /Applications/Atom.app/Contents/Resources/app/src/atom-environment.js:1263:35
```

### Commands

```
     -0:21.2.0 api-workbench:new-project (input.hidden-input)
```

### Non-Core Packages

```
api-workbench 0.8.46 
busy-signal 1.4.3 
intentions 1.1.5 
linter 2.2.0 
linter-ui-default 1.6.5 
```

**Comment by ddenisenko** [comment]
Confirmed in Atom 1.19.1, thank you.

**Comment by dreamflyer** [comment]
will be published in v0.8.47.
